pkg/model/deployment.go

```diff
@@ -52,13 +59,17 @@ func DeploymentName(backstageName string) string {
 // BackstageContainerIndex returns the index of backstage container in from deployment.spec.template.spec.containers array
 func BackstageContainerIndex(bsd *appsv1.Deployment) int {
 	for i, c := range bsd.Spec.Template.Spec.Containers {
-		if c.Name == "backstage-backend" {
+		if c.Name == BackstageContainerName() {
 			return i
 		}
 	}
 	return -1
 }
 
+func BackstageContainerName() string {
```

Comment:
Since this returns the same string, could we use a simple constant instead (for better compiler optimization)?

Reply:
It used to be, I converted it in purpose.
The reason is that we might want to change the way we recognize backstage container name, for example put it as a deployment's lable or so...